lib/index.ts

```diff
@@ -131,6 +130,44 @@ class LogService {
     }
   }
 
+  addLogger(
+    loggerScopeFunction: (config: InitialConfig) => LoggerPlugin,
+    pluginConfig: PluginConfig) {
+    if (typeof loggerScopeFunction !== 'function')
+      return;
+
+    const loggerOptions = loggerScopeFunction.call(this, this.globalConfig);
+
+    if (loggerOptions == null)
+      return;
+
+    const loggerKeys = Object.keys(loggerOptions);
```

Comment:
Don't return let the flow go, and add a nullish operator in object.keys to pass an empty object if null:
`Object.keys(loggerOptions ?? {});`